My first pull request on Hello Microverse project

paulsaenzsucre opened the pull request:
In this pull request a made these changes:

- Write an index.html file with a header.
- Write a styles.css file with a class for styling the header.
- Write a descriptive README file.
- Include a MIT license.

Also, did the following activities:

- Setup the linters for my project.
- Look for and fix linter errors.
- Create a new branch for coding.
- Do frequents commits with incremental changes.

paulsaenzsucre committed:
Create linters.yml

Adding linters to check the format

paulsaenzsucre committed:
Update README.md file

paulsaenzsucre committed:
Add index.html

paulsaenzsucre committed:
Add styles.css file and link to the index.html

paulsaenzsucre committed:
Add .gitignore

codecaiine commented on .github/workflows/linters.yml:
```diff
@@ -0,0 +1,56 @@
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install --save-dev hint@7.x
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
+      - name: Webhint Report
+        run: npx hint .
+  stylelint:
+    name: Stylelint
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  nodechecker:
+    name: node_modules checker
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check node_modules existence
+        run: |
+          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
```
- [x] Good job so far :+1:   I noticed that your linters are not correctly set up:cry: Files stylelintrc.json, .hintrc are missing in your setup. Please follow this [Linters HTML and CSS](https://github.com/microverseinc/linters-config/tree/master/html-css). Linters are very important because they will help to become a better developer by fixing code quickly and having your code syntax correct. :+1: 

![Screenshot from 2022-10-25 09-17-56](https://user-images.githubusercontent.com/6615394/197735431-f4df3e1a-1751-4b11-8841-ad2d253a692d.png)

codecaiine commented on README.md:
```diff
@@ -1 +1,22 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Hello Microverse!.
+
+
+## Built With
+
+- HTML/CSS
+- Git/Github/Linter
+
+## Authors
+
+👤 **Paul Sáenz Sucre**
+
+- GitHub: [@elalfildecolor](https://github.com/elafildecolor)
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```
- [x] Please add a descriptive README file to your project by using [this template](https://github.com/microverseinc/readme-template) and customize it. Please make sure to add and keep only the main header section and do not keep them empty. Also add a license in order to make it more professional :+1: 
- [x] Your project doesn't have a license. I would like to recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish and add the licence link to section in your readme dedicated for it in order to keep it more professional :+1:

codecaiine reviewed (changes requested):
Hi @elafildecolor  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good use of github flow :heavy_check_mark: 
- Good project implementation :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] [OPTIONNAL] Please make sure that your pull Request has a descriptive summary with more details about the changes. Please always make your PR professional, not leave your PR summary empty before submitting :+1: 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please in your future project make your branch name more descriptive instead of feature :smile:  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

paulsaenzsucre committed:
Add linters files

- .hintrc

.stylelintrc.json

package.json

package-lock.json

paulsaenzsucre committed:
Add MIT License

A short and simple permissive license.

paulsaenzsucre committed:
Beautify README

- Add title.

- Add better description.

- Add live demo link.

Add getting started section.

Add contributing, show your support and acknowledgments sections.

Add license section with a link to the license.

paulsaenzsucre committed:
Add a header tag  in index.html file

paulsaenzsucre committed:
Add a class for header styling

- Making background color blue.

- Making foreground color white.

paulsaenzsucre committed:
Fixing a typo in README file

topeogunleye reviewed (approved):
### Hi @elafildecolor, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
![congratulations](https://user-images.githubusercontent.com/52934128/163691476-087b4b9c-825b-40a3-85a1-d1bde44ffa53.gif)
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
 
## [Optional] suggestions:
*  Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
